libraries/objectDefaultFiles/object.js

```diff
@@ -2280,8 +2293,8 @@
                     name: processTitle,
                     numStopsRequired: options.numStopsRequired // if included, stop will need to be called this many times
                 }
-            });
-        };
+            })
+        }
```

Comment:
nit: missing semicolons here and throughout the file in the new functions you've added (you can check the code on github or do `eslint .` locally (if you've installed eslint) to see where the linting system is complaining)